tests/all_tests_v2.nim

```diff
@@ -12,7 +12,7 @@ import
   ./v2/test_jsonrpc_waku,
   ./v2/test_peer_manager,
   ./v2/test_web3, # TODO  remove it when rln-relay tests get finalized
-  ./v2/test_waku_rln_relay,
+  # ./v2/test_waku_rln_relay,
```

Comment:
@staheri14, you may find it interesting that updating submodules causes [CI to fail with a "connection refused" `TransportOsError`](https://github.com/status-im/nim-waku/pull/594/checks?check_run_id=2771905449#step:6:3003) on the local `EthClient` address opened in `test_waku_rln_relay`. This only happens during CI, not when testing on a local machine. I've disabled the RLN tests for now, though https://github.com/status-im/nim-waku/pull/602 should similarly work around the issue.

Reply:
Sorry, missed this comment of yours, I think now that the rln compilation is optional, this problem should not happen, right?

Reply:
Right! I revered this change once I merged the rln optional stuff.